### PR TITLE
add synonym to MONDO:0004719 'ichthyosis (disease)'

### LINKED ISSUE #2081: Add Synonym: Ichthyosis (disease)
**Mondo term (ID and Label)**
Ichthyosis (disease) (`MONDO_0016444`)

**Synonym to be added**
- xeroderma  
- xerodermia

**Synonym subtype: ie, broad/exact/narrow/related**
Exact

**Database cross reference for the synonym** Such as PubMed ID (in the format PMID:XXXXXX) or a cross-reference to another ontology, like OMIM or Orphanet.
- `SCTID:52475004`  
- `OMOP:4198562`

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)
https://orcid.org/0000-0002-8169-9049

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
and add logical def to 'xeroderma of eyelid'
fixes #2081

@pnrobinson and @cmungall can you two (or one of you) review this one?

Tiffany asked for xeroderma to be an exact synonym of ichthyosis (disease) (according to OMOP and Snomed) and from my google search, this seems right: https://www.google.com/search?q=xeroderma&oq=xeroderma&aqs=chrome.0.69i59j0l7.398j0j7&sourceid=chrome&ie=UTF-8
image

I added the logical def to 'xeroderma of eyelid' - if xeroderma is an exact syn of ichthyosis (disease), then this seemed appropriate.

Thanks!

### CODE REVIEW

**Comment by pnrobinson** [comment]
These are not synonyms!

Please add this definition for ichthyosis

The ichthyoses are disorders of cornification that are characterized by visible scaling and/or hyperkeratosis of most or all of the skin. Inherited ichthyoses, defined as the generalized form of Mendelian disorders of cornification, affect most or all of the skin. This etiologically and phenotypically heterogenous group of conditions is caused by mutations in various different genes important for keratinocyte differentiation and epidermal barrier function. Acquired forms of ichthyosis can be observed with certain autoimmune, inflammatory, metabolic, endocrine, or infectious diseases or with malignancies.

PMID: 22739337

comment:   Epidermal keratinocytes undergo a unique form of terminal differentiation and programmed cell death known as cornification. Cornification leads to the formation of the outermost skin barrier, i.e. the cornified layer, as well as to the formation of hair and nails.  PMID: 23792051

syn fish scale disease or fish skin disease

Probably use this article to develop the hierarchy: https://pubmed.ncbi.nlm.nih.gov/20643494/

**Comment by pnrobinson** [comment]
Also, I do not think that xeroderma should be modeled as a disease (=dry skin). This is a phenotypic feature of many diseases and can commonly occurs as an isolated feature, e.g., in New England winters. The disease would include the etiology, so in many cases would be something like "Winter rash". 
The HPO includes terms for both dry skin, which is clearly a feature, as well as for Ichthyosis HP:0008064 (An abnormality of the skin characterized the presence of excessive amounts of dry surface scales on the skin resulting from an abnormality of keratinization.). In this case, the HPO term refers to the cornification disorder observed in the skin, and is one of the features of the ichtyosis diseases. The diseases have multiple other features: https://hpo.jax.org/app/browse/disease/ORPHA:313
There is no other word in English that captures the skin feature, possible we should chance the label to ichtyotic skin though

**Comment by nicolevasilevsky** [comment]
Thanks @pnrobinson!

I will close this PR. I created a new PR to add a new def:
https://github.com/monarch-initiative/mondo/pull/2113
